awxkit/awxkit/cli/resource.py

```diff
@@ -44,6 +49,56 @@
     (v, k) for k, v in DEPRECATED_RESOURCES.items()
 )
 
+EXPORTABLE_RESOURCES = [
+    'users',
+    'organizations',
+    'teams',
+    'credential_types',
+    'credentials',
+    # 'notification_templates',
+    # 'projects',
+    # 'inventory',
+    # 'job_templates',
+    # 'workflow_job_templates',
+]
+
+NATURAL_KEYS = {
+    'user': ('username',),
+    'organization': ('name',),
+    'team': ('organization', 'name'),
+    'credential_type': ('name', 'kind'),
+    'credential': ('organization', 'name', 'credential_type'),
+    'notification_template': ('organization', 'name'),
+    'project': ('organization', 'name'),
+    'inventory': ('organization', 'name'),
+    'job_template': ('name',),
```

Comment:
I changed this, it may be `('organization', 'name')`, read-oriented, but `('project.organization', 'name')` in another write-oriented manner. I hope I did not screw everything up.

Reply:
Not a big deal, your work is on my radar.  I'm going to have to come up with a fallback scheme for missing identifiers anyway, not just for this but also the workflow template nodes.

Reply:
> I'm going to have to come up with a fallback scheme for missing identifiers anyway

I feel like I might be working on something very similar with the AWX collection right now. Depending on your exact meaning. But that's in a totally different place, with significant differences in how the parameters are received.